app/src/main/java/org/sopt/and/ui/home/PopularContentLazyList.kt

```diff
@@ -0,0 +1,51 @@
+package org.sopt.and.ui.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun ContentLazyList(imgList: List<Int>) {
+    LazyRow(
+        modifier = Modifier.padding(bottom = 10.dp),
+        contentPadding = PaddingValues(end = 16.dp),
+        horizontalArrangement = Arrangement.spacedBy(12.dp)
+    ) {
+        items(imgList,
+            key = { index -> index },
+            contentType = { imgList -> imgList }) { img ->
+            Content(img = img)
+        }
+    }
+}
+
+@Composable
+fun Content(img: Int) {
```

Comment:
```suggestion
fun Content(
    @DrawableRes image: Int
) {
```
정도는 어떨까요? 

- Int 타입만 보면 어떤 것이 들어가야할지 잘 모를 수 있아요!
- 어노테이션으로 어떤 값들이 들어가야하는지 알려주는게 좋습니다.

Reply:
확인했습니다!